WordPress/Classes/Services/PostService.m

```diff
@@ -192,7 +192,12 @@ - (void)uploadPost:(AbstractPost *)post
                     if (success) {
                         success(postInContext);
                     }
-                } failure:failure];
+                } failure:^(NSError *error){
+                    // Sergio Estevao: even if media fails to attach we are answering with success because the post uploaded sucessfull and the only thing that failed what attaching the media to it.
```

Comment:
There's a typo here, `he only thing that failed what` should be `he only thing that failed was`.